BedrockServer.cpp

```diff
@@ -333,10 +333,23 @@ void BedrockServer::sync()
 
             // Process any activity in our plugins.
             AutoTimerTime postPollTime(postPollTimer);
+            auto start = STimeNow();
             _postPollCommands(fdm, nextActivity);
+            auto first = STimeNow();
             _syncNode->postPoll(fdm, nextActivity);
+            auto second = STimeNow();
             _syncNodeQueuedCommands.postPoll(fdm);
+            auto third = STimeNow();
             _completedCommands.postPoll(fdm);
+            auto finish = STimeNow();
+
+            if ((finish - start) > 1'500'000) {
+                SINFO("[diag] Post poll took " << (finish - start) << "us, breakdown: "
```

Comment:
tag these and others `[performance]` so they skip elasticsearch? otherwise this could be really noisy